src/Approximations/overapproximate.jl

```diff
@@ -714,6 +714,22 @@ end
 # Overapproximation of Sparse Polynomial Zonotopes
 # ================================================
 
+#=
+Given center, (dependent) generator matrix and exponent matrix of a (simple) sparse polynomialzonotope,
+compute thew new center and generator matrix of its zonotope overapproximation.
+=#
+function _zonotope_overapprox(c, G, E)
+    cnew = copy(c)
+    Gnew = copy(G)
+    @inbounds for (j, g) in enumerate(eachcol(G))
+        if all(iseven, E[:, j])
+            cnew .+= 0.5 * g
+            Gnew[:, j] ./= 2
```

Comment:
This is somewhat inconsistent. 
```suggestion
            cnew .+= g / 2
            Gnew[:, j] ./= 2
```